core/src/main/java/org/apache/iceberg/util/LockManager.java

```diff
@@ -17,14 +17,14 @@
  * under the License.
  */
 
-package org.apache.iceberg.aws.glue;
+package org.apache.iceberg.util;
```

Comment:
why is this in the `util` package? If we want to properly support it, it feels more nature to me to add this to `iceberg-api`.

Reply:
done